addons/account/models/chart_template.py

```diff
@@ -560,12 +569,16 @@ def _get_chart_template_data(self, template_code):
                 key=lambda i: TEMPLATE_MODELS.index(i[0]) if i[0] in TEMPLATE_MODELS else 1000
             ):
                 for func in funcs:
-                    data = func(self, template_code)
+                    data = func(self.with_context(lang='en_US') if model in TEMPLATE_MODELS and self.env[model]._fields['name'].translate else self,
+                                template_code)
```

Comment:
Not sure that we need to check `self.env[model]._fields['name'].translate`, the fact that it is in `TEMPLATE_MODEL` should be enough

```suggestion
                self_lang = self
                if model in TEMPLATE_MODELS and self.env[model]._fields['name'].translate:
                    self_lang = self.with_context(lang='en_US')
                for func in funcs:
                    data = func(self_lang, template_code)